playbook-website/app/javascript/site_styles/_visual_guidelines.scss

```diff
@@ -42,4 +43,69 @@
     .nine { margin-left: 400px; }
     .ten { margin-left: 450px; }
   }
+
+  [class^=pb_title_kit][class*=_3].ls_tightest {
+      letter-spacing: $lspace_tightest !important;
+    }
+
+  [class^=pb_title_kit][class*=_3].ls_tighter {
+      letter-spacing: $lspace_tighter !important;
+    }
+
+  [class^=pb_title_kit][class*=_3].ls_tight{
+    letter-spacing: $lspace_tight !important;
+  }
+
+  [class^=pb_title_kit][class*=_3].ls_normal{
+    letter-spacing: $lspace_normal !important;
+  }
+
+  [class^=pb_title_kit][class*=_3].ls_loose{
+    letter-spacing: $lspace_loose !important;
+  }
+
+  [class^=pb_title_kit][class*=_3].ls_looser{
+    letter-spacing: $lspace_looser !important;
+  }
+
+  [class^=pb_title_kit][class*=_3].ls_loosest{
+    letter-spacing: $lspace_loosest !important;
+  }
+
+  [class^=pb_title_kit][class*=_3].ls_super_loosest{
+    letter-spacing: $lspace_super_loosest !important;
+    text-align: center !important;
+  }
```

Comment:
If you are demonstrating the tokens then you should use raw text, not kits.

Also can you move these type settings to their own sass file and import it here instead? That will keep this file clean. I gave Caitlin simular feedback so you might be able to glean off what she did.

Reply:
Since that's demonstrating letter spacing, I wanted to use Title 3 for better readability. Do you mean I should use 'Body' instead?

And will do!

Reply:
Just use a raw div and classes. We are showing how to use the tokens not the kits.